portfolio/src/main/webapp/script.js

```diff
@@ -12,28 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/* References: https://www.w3schools.com/howto/howto_js_slideshow.asp
- */
-
-const sleep = (milliseconds) => {
-  return new Promise(resolve => setTimeout(resolve, milliseconds))
-}
-
 var slideIndex = 1;
-sleep(50).then(() => {
-  showSlides(slideIndex);
-})
+showSlides(slideIndex);
 
-// Next/previous controls
+// Next and previous controls
 function plusSlides(n) {
   showSlides(slideIndex += n);
 }
 
-// Dot control
+// Allows changing slides through the dots
 function currentSlide(n) {
   showSlides(slideIndex = n);
 }
 
+// Updated slides based on previous/next click or dot click
 function showSlides(n) {
```

Comment:
Functions should have a JSDoc, so the comment format must be /** */. Example:

/**
 * Multiple lines of JSDoc text are written here,
 * wrapped normally.
 * @param {number} arg A number to do something to.
 */
function doSomething(arg) { … }

Reply:
Added comments